platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/signed/SignedStateValidationData.java

```diff
@@ -23,29 +23,25 @@
 
 /**
  * Basic record object to carry information useful for signed state validation.
- * @param round the minimum round to be considered a valid state
- * @param numberOfConsensusEvents the count of all consensus events in history from an earlier state
- * @param consensusTimestamp The consensus timestamp from an earlier state
- * @param lastTransactionTimestamp The last transaction timestamp from an earlier state
- * @param addressBookHash The address book hash value for the current address book (mostly used for diagnostics).
- * @param consensusEventsRunningHash The running hash of the consensus event hashes throughout history
- * @param epochHash The epoch hash from an earlier state.
+ *
+ * @param round
+ * 		the minimum round to be considered a valid state
+ * @param consensusTimestamp
+ * 		The consensus timestamp from an earlier state
+ * @param addressBookHash
+ * 		The address book hash value for the current address book (mostly used for diagnostics).
+ * @param consensusEventsRunningHash
+ * 		The running hash of the consensus event hashes throughout history
+ * @param epochHash
+ * 		The epoch hash from an earlier state.
```

Comment:
Your intellij configuration seems to be auto-formatting this differently than other people's configuration.

Reply:
I don't know what to do about this:
- spotless was supposed to fix this
- the intellij config is in the repo, shouldn't we be using whatever is in the repo?